examples/Makefile

```diff
@@ -9,8 +9,9 @@ install: install-observability install-argo
 	@echo "-----------------------------------"
 	@echo "Create Namespace and install Keptn-lifecycle-toolkit"
 	@echo "-----------------------------------"
-	kubectl apply -f https://github.com/keptn/lifecycle-toolkit/releases/download/v0.5.0/manifest.yaml #x-release-please-version
-	kubectl wait --for=condition=available deployment/lifecycle-operator -n keptn-lifecycle-toolkit-system --timeout=300s
+	helm repo add keptn-lifecycle-toolkit https://charts.lifecycle.keptn.sh
+	helm repo update
+	helm upgrade --install keptn-lifecycle-toolkit keptn-lifecycle-toolkit/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
```

Comment:
please use klt to name the release, by default we add the chart name as a prefix to everything, if it's too long k8s will scream

Reply:
ok, so you mean something like:

`helm upgrade --install klt keptn-lifecycle-toolkit/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait` ?

Reply:
yes please :)